pkg/api/configsync/v1beta1/gitconfig.go

```diff
@@ -25,9 +25,11 @@ type Git struct {
 	// repo is the git repository URL to sync from. Required.
 	Repo string `json:"repo"`
 
-	// branch is the git branch to checkout.
-	// The field is deprecated. Use `revision` instead.
-	// If both `branch` and `revision` are defined, `revision` takes precedence over `branch`.
+	// branch is the git branch to sync from.
+	// The field is consolidated with the `revision` field.
+	// If `revision` is not specified, `branch` will be used.
+	// If both `branch` and `revision` are specified, `revision` takes precedence over `branch`.
+	// Recommend using `revision`.
```

Comment:
I suggest using a complete sentence for this last line. For example:

"It is recommended to use the `revision` field"

Reply:
done

Reply:
Please update both API versions and the CRDs

Reply:
oops, thanks for catching this.